spyder/plugins/editor/plugin.py

```diff
@@ -251,20 +258,25 @@ def enable_tabwidth_spin(index):
                 tabwidth_spin.plabel.setEnabled(False)
                 tabwidth_spin.spinbox.setEnabled(False)
 
-        indent_chars_box.combobox.currentIndexChanged.connect(enable_tabwidth_spin)
+        indent_chars_box.combobox.currentIndexChanged.connect(
+            enable_tabwidth_spin)
 
         tab_mode_box = newcb(_("Tab always indent"),
-                      'tab_always_indent', default=False,
-                      tip=_("If enabled, pressing Tab will always indent,\n"
-                            "even when the cursor is not at the beginning\n"
-                            "of a line (when this option is enabled, code\n"
-                            "completion may be triggered using the alternate\n"
-                            "shortcut: Ctrl+Space)"))
+                             'tab_always_indent', default=False,
+                             tip=_("If enabled, pressing Tab will always "

```

Comment:
Again, another prime opportunity for a hanging indent to avoid the really messy and hard to read ``tip`` text.